### PR TITLE
feat: display flag icons based on listeners languages

### PR DESCRIPTION
**Pull request by NoeliaEle**
- Default flag icon is DE
- Every listener has DE as a language. To be updated if we recruit listeners that don't speak DE, we would also need to update all the coaches in Contentful and add DE to those who speak german

### CODE REVIEW

File: app/utils/getFlagCodes.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    ru: "RU",
+    de: "DE",
+    en: "GB",
+    esp: "ES",
+    nl: "NL",
+    nor: "NO",
+    pol: "PL",
+    por: "PT",
+  };
+  const result = [];
```
Comment (HerrBertling) [inline app/utils/getFlagCodes.ts:13]: You could directly set `const result = [“DE”]`, skip the logic below and do the `forEach` loop then. If necessary, any double entries could be removed with a `Set`, but from what I get, that’s currently not the case, right?

Comment (NoeliaEle) [reply app/utils/getFlagCodes.ts:13]: Let me try that. Some listeners have "DE" already, some not, so I think we would need the `Set`

Comment (HerrBertling) [reply app/utils/getFlagCodes.ts:13]: Yeah, you can return `[...new Set(whatever)]` :)

**Review by HerrBertling — APPROVED** [review]
:tada:

**Pull request merged**